crates/solidity/testing/snapshots/cst_output/TupleDeconstructionStatement/empty/generated/0.4.11-success.yml

```diff
@@ -0,0 +1,23 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ // TODO: Tuple expressions can't be empty                                        │ 0..41
+  2  │ () = ();                                                                         │ 42..50
```

Comment:
This should have failed. Did I miss something?

Reply:
This is parsed incorrectly on `main` atm, as we can't express "separated list of optional items, but having at least 1 item" in terms of the "separated" grammar node, short of `(elem (separator elem )*)`.

We should fix it and I'll open an issue for it; wanted to leave a TODO note explaining that the current test case success is not actually expected.

Reply:
I see. I suggest adding the issue # to this TODO comment here, if the bug already exists on main.

Reply:
Added in https://github.com/NomicFoundation/slang/pull/667/commits/23f88047d61fb5b6b4a459f643029e4069e83aba